uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/document.controller.spec.js

```diff
@@ -38,20 +40,34 @@ define([
       it('it initializes the document\'s file URL', function () {
         expect($scope.fileUrl).toBe(expectedUrl);
       });
+
+      describe('when the document id is updated', function () {
+        beforeEach(function () {
+          $scope.document.id += 1;
```

Comment:
is document.id String or Number?

Reply:
It's a string, but that doesn't matter, it changes regardless of whether it's a string or a number and $scope.watch detects that.

Reply:
Not really. If id is "1" I expect it to be "2" according to the code. However it will be "11". If I then compare IDs that could be an issue. Please change to something like:

$scope.document.id = (+$scope.document.id + 1).toString();